code/obj/item/gun/kinetic.dm

```diff
@@ -1676,7 +1676,7 @@ ABSTRACT_TYPE(/obj/item/gun/kinetic/single_action)
 
 
 /datum/component/holdertargeting/smartgun/nukeop/is_valid_target(mob/user, mob/M)
-	return ..() && !istype(M.get_id(), /obj/item/card/id/syndicate)
+	return ..() && !(istype(M.get_id(), /obj/item/card/id/syndicate) || isnukeopgunbot(M) || istype(M, /mob/living/critter/robotic/sawfly))
```

Comment:
you'll probably want to do the isnukeopgunbot(x) define here. There's not a define for sawflies yet, (#11043 fixes that), but once this gets merged I'll add the define check as well.

Reply:
update: I may or may not be blind